dataline-workers/src/test/java/io/dataline/workers/TestConfigHelpers.java

```diff
@@ -0,0 +1,119 @@
+/*
+ * MIT License
+ *
+ * Copyright (c) 2020 Dataline
+ *
+ * Permission is hereby granted, free of charge, to any person obtaining a copy
+ * of this software and associated documentation files (the "Software"), to deal
+ * in the Software without restriction, including without limitation the rights
+ * to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+ * copies of the Software, and to permit persons to whom the Software is
+ * furnished to do so, subject to the following conditions:
+ *
+ * The above copyright notice and this permission notice shall be included in all
+ * copies or substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ * FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+ * SOFTWARE.
+ */
+
+package io.dataline.workers;
+
+import io.dataline.commons.json.JsonUtils;
+import io.dataline.config.Column;
+import io.dataline.config.DataType;
+import io.dataline.config.DestinationConnectionImplementation;
+import io.dataline.config.Schema;
+import io.dataline.config.SourceConnectionImplementation;
+import io.dataline.config.StandardSync;
+import io.dataline.config.StandardSyncInput;
+import io.dataline.config.State;
+import io.dataline.config.Table;
+import java.util.Collections;
+import java.util.Map;
+import java.util.UUID;
+import org.apache.commons.lang3.tuple.ImmutablePair;
+
+public class TestConfigHelpers {
+  private static final String CONNECTION_NAME = "favorite_color_pipe";
+  private static final String TABLE_NAME = "user_preferences";
+  private static final String COLUMN_NAME = "favorite_color";
+  private static final long LAST_SYNC_TIME = 1598565106;
+
+  public static ImmutablePair<StandardSync, StandardSyncInput> createSyncConfig() {
+    final UUID workspaceId = UUID.randomUUID();
+    final UUID sourceSpecificationId = UUID.randomUUID();
+    final UUID sourceImplementationId = UUID.randomUUID();
+    final UUID destinationSpecificationId = UUID.randomUUID();
+    final UUID destinationImplementationId = UUID.randomUUID();
+    final UUID connectionId = UUID.randomUUID();
+
+    final String sourceConnection =
+        JsonUtils.toJson(
```

Comment:
You need to get master into your branch. JsonUtils doesn't exist anymore. `Jsons.serialize(xxx)`